justfile

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
```

Comment:
With just, you can add a default rule which runs when no args have been specified. I like it because it gives it more discoverability.

I did it with Everett:

https://github.com/willkg/everett/blob/main/justfile#L3-L4

Also, we can add a comment before each rule and give it a one-line help text for what it does. That shows up when you do `just --list`:

```
> just --list
Available recipes:
    build  # Build files for relase
    clean  # Wipe devenv and build artifacts
    devenv # Build a development environment
    docs   # Runs cog and builds Sphinx docs
    format # Format files
    lint   # Lint files
    test   # Run tests, linting, and static typechecking
```